web/src/app/modules/shared/components/presentation/modal/modal.component.html

```diff
@@ -0,0 +1,122 @@
+<clr-modal [(clrModalOpen)]="opened" [clrModalStaticBackdrop]="false" [clrModalClosable]="true" [clrModalSize]="size">
+  <h3 class="modal-title">
+    <app-view-title [views]="title"></app-view-title>
+  </h3>
+  <div class="modal-body">
+    <app-view-container [view]="body"></app-view-container>
+    <form clrForm *ngIf="form" (ngSubmit)="onFormSubmit()" [formGroup]="formGroup">
+      <div *ngFor="let field of form.fields; trackBy: trackByFn">
+        <ng-container [ngSwitch]="field.type">
```

Comment:
The code / html that renders forms should be in a separate component.

Reply:
Form is now extracted from card and summary components. This cannot be applied to the stepper because there is validation between each step and it is simpler to let the clarity code do the lifting.